page-object/pages/DatasetsPage.ts

```diff
@@ -3,13 +3,26 @@ import { BasePage } from "../BasePage";
 
 export class DatasetsPage extends BasePage{
     datasetsPageLocator: any;
+    datasetsUpvoteBtn: any;
+    datasetsUpvoteNmb: any;
+    datasetsPageLocatorNav: any;
 
     constructor(page){
         super(page);
         this.datasetsPageLocator = page.locator("//a[@data-click-log-id='nav-con-datasets']");
+        this.datasetsPageLocatorNav = page.locator(`//a[@data-click-log-id="datasets"]`);
+        this.datasetsUpvoteBtn = page.locator(`(//button[@data-testid="upvotebutton__upvote"])[1]`)
+        this.datasetsUpvoteNmb = page.locator(`(//span[@class="sc-bBkKde sc-fccqSk epgXIi kpoHlg"])[1]`)
     }
```

Comment:
Maybe call these two elements as 'previewFirstUpvoteBtn' and 'previewFirstUpvoteNumber'? Just for more clear understanding of which type of upvote is happening in test